draft-ietf-rats-corim.md

```diff
@@ -843,68 +843,70 @@ The types defined for a group identified are UUID and variable-length opaque byt
 {::include cddl/group-id-type-choice.cddl}
 ~~~
 
-##### Measurements
+##### Claims
+
+A Claim may either be a measurement that can be directly compared with attestation evidence, or it can be an endorsement of a quality that is not directly comparable with evidence.
```

Comment:
Trying to recover a suggestion that got lost in the discussion.
```suggestion
A Claim represents a single piece of actual or reference state.
A Claim may either be a measurement that can be directly compared with attestation evidence, or it can be an endorsement of a quality that is not directly comparable with evidence.
```
There was a further suggestion to s/measurement/reference value/ in L848